modules/system/console/WinterMirror.php

```diff
@@ -201,7 +197,49 @@ protected function mirror($src, $dest)
             }
         }
 
-        symlink($src, $dest);
+        foreach ($this->option('ignore') as $ignore) {
+            if (preg_match($ignore, $src)) {
+                $this->warn('ignoring: ' . $src);
+                return false;
+            }
+        }
+
+        if (!$this->option('copy')) {
+            symlink($src, $dest);
+            $this->info('Linked: ' . $dest);
+            return true;
+        }
+
+        if (is_dir($src) && !is_dir($dest)) {
+            mkdir($dest, 0755, true);
+        }
+
+        if (is_file($src)) {
+            !is_dir(dirname($dest)) && mkdir(dirname($dest), 0755, true);
```

Comment:
Should these permissions come from the `Winter\Storm\Filesystem\Filesystem->getFolderPermissions() / getFilePermissions()`? Since those come from `cms.defaultMask.file` and `cms.defaultMask.folder`?